internal/core/swrenderer.rs

```diff
@@ -27,6 +27,8 @@ pub use draw_functions::{PremultipliedRgbaColor, TargetPixel};
 
 type DirtyRegion = PhysicalRect;
 
+pub use crate::lengths::{PhysicalLength, PhysicalSize};
```

Comment:
3 lines earlier there is another `pub use` i think it makes sense to keep them together.

Reply:
Agreed.